src/components/CommentTeaser/CommentTeaser.js

```diff
@@ -61,7 +61,11 @@ export const CommentTeaser = ({
         lineClamp ? merge(styles.clamp, { WebkitLineClamp: lineClamp }) : {}
       )}
     >
-      {renderMdast(content, schema)}
+      {renderMdast(
+        content,
+        schema,
+        { MissingNode: ({ children }) => <span>{children}</span> }
```

Comment:
We should use the same `MissingNode` as in the main comment component: https://github.com/orbiting/styleguide/blob/2adf2bb22abcc83287873cb50ad0cdcbfcce3bab/src/components/Comment/Comment.js#L40

Reply:
@lukasbuenger I'll take care of this.